cmd/openfga/openfga_test.go

```diff
@@ -0,0 +1,404 @@
+//go:build functional
+// +build functional
```

Comment:
This is a Go build tag, which means that this file will only get included in the test build if you pass the `-tags=functional` flag to `go test`. The idea I was going for is that, by default, functional tests don't run. You have to specify the build tag to get the functional tests to run.